src/tests/testing/model/GOTestOrganModel.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+#include <iostream>
+
+#include "GOTest.h"
+#include "GOTestCollection.h"
+#include "GOTestException.h"
+#include "GOTestOrganModel.h"
+
+#include "GOOrganController.h"
+#include "config/GOConfig.h"
+#include "model/GOWindchest.h"
+
+GOTestOrganModel::~GOTestOrganModel() {}
+
+void GOTestOrganModel::run() {
+  wxString message;
+  // Set OrganModel Modified
+  this->controller->SetOrganModelModified(true);
+  message = "Is Organ Modified value is not True";
```

Comment:
Do note that all such kinds of wxString assignments will be at the mercy of the current locale encoding. Generally it of course should be safe with pure English characters, but using the appropriate macros is still recommended.